lib/obj.js

```diff
@@ -206,7 +206,12 @@ function loadObj(objPath) {
                 var nx = parseFloat(result[1]);
                 var ny = parseFloat(result[2]);
                 var nz = parseFloat(result[3]);
-                var normal = Cartesian3.fromElements(nx, ny, nz, scratchCartesian);
+                var normal;
+                if(nx === 0 && ny === 0 && nz === 0) {
+                    normal = Cartesian3.UNIT_X;
+                } else {
+                    normal = Cartesian3.fromElements(nx, ny, nz, scratchCartesian);
+                }
```

Comment:
Preferably obj2gltf should do as little processing/checking on the file as possible and leave that to gltf-pipeline. I have similar checking code in https://github.com/AnalyticalGraphicsInc/gltf-pipeline/pull/240, which should be sufficient.